### PR TITLE
Update workflows

### PR DESCRIPTION
**Pull request by cicdguy**
Update package to conform with standardized workflows for R projects in IE.
Closes https://github.com/insightsengineering/rlistings/issues/37

### CODE REVIEW

**Review by edelarua — APPROVED** [review]
Lgtm if others approve!

**Review by Melkiades — APPROVED** [review]
I read the code and run checks and tests locally and all seems to work perfectly for me. I added an import as it was giving notes in checks. Lgtm!

File: R/rlistings.R [hunk 1]
```diff
@@ -31,10 +32,10 @@ setOldClass(c("MatrixPrintForm", "list"))
 #'
 #' dat <- ex_adae
 #'
-#' lsting <- as_listing(dat[1:25,], key_cols = c("USUBJID", "AESOC")) %>%
-#'     add_listing_col("AETOXGR") %>%
-#'     add_listing_col("BMRKR1", format = "xx.x") %>%
-#'     add_listing_col("AESER / AREL", fun = function(df) paste(df$AESER, df$AREL, sep = " / "))
+#' lsting <- as_listing(dat[1:25, ], key_cols = c("USUBJID", "AESOC")) %>%
```
Comment (gmbecker) [inline R/rlistings.R:35]: @cicdguy we already ran rlistings through the linter, why is this PR changing essentially every line of code, again?

**Review by gmbecker — CHANGES_REQUESTED** [review]

File: R/rlistings.R (hunk 1 above)
Comment (gmbecker) [reply R/rlistings.R:35]: Also even if we are going to lint it (again) taht should be a separate PR shouldn't it?

Comment (gmbecker) [reply R/rlistings.R:35]: I don't know, @edelarua did it the last time around.

Comment (edelarua) [reply R/rlistings.R:35]: We didn't use `styler` last time. Only corrected the lint.

**Comment by Melkiades** [comment]
@cicdguy will this also fix the broken link to the rlistings website?

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks @cicdguy !

**Pull request merged**